packages/entity-database-adapter-knex/src/PostgresEntityDatabaseAdapter.ts

```diff
@@ -42,18 +42,45 @@ export default class PostgresEntityDatabaseAdapter<
   }
 
   protected async fetchManyWhereInternalAsync(
-    queryInterface: any,
+    queryInterface: Knex,
     tableName: string,
     tableColumns: readonly string[],
-    tableValueValues: (readonly any[])[],
```

Comment:
I think tableTuples or tableValuesList or tableValuesArray is clearer and more accurate.

It previously was a list where each item was a table value: tableValues.
Now it's a list where each item is table values: "tableValuess" so to speak. But I don't think it becomes ValueValues.